docs/UserGuide.adoc

```diff
@@ -212,18 +212,18 @@ The `redo` command fails as there are no `undo` commands executed previously.
 
 * `delete 1` +
 `clear` +
-`undo` (reverses the `clear` command) +
-`undo` (reverses the `delete 1` command) +
-`redo` (reapplies the `delete 1` command) +
-`redo` (reapplies the `clear` command) +
+`ud` (reverses the `clear` command) +
+`ud` (reverses the `delete 1` command) +
+`rd` (reapplies the `delete 1` command) +
+`rd` (reapplies the `clear` command) +
 // end::undoredo[]
 
-=== Clearing all entries : `clear`
+=== Clearing all entries : `clear` or `c`
 
 Clears all entries from the address book. +
 Format: `clear`
 
-=== Exiting the program : `exit`
+=== Exiting the program : `exit` or `ex`
```

Comment:
I think that `ex` isn't a very intuitive alias; maybe change to `q` for quit? (Because `q` is more commonly used for exiting a program, especially in Linux. This will be useful and more intuitive for Linux users, whom we have to cater to under our project requirements)